docs/instrumentation/jvmagent.adoc

```diff
@@ -40,7 +40,7 @@ will override the environment variables. The property names are:
 | HAWKULAR_APM_USERNAME | The username to use with the REST client when communicating with the APM server. The default defined in the _setenv.sh_ script is *_jdoe_*.
 | HAWKULAR_APM_PASSWORD | The password to use with the REST client when communicating with the APM server. The default defined in the _setenv.sh_ script is *_password_*.
 | HAWKULAR_APM_CONFIG_REFRESH | Optional business transaction configuration refresh interval (in seconds).
-| HAWKULAR_APM_LOG_LEVEL | The logging level to use when reporting client collector information to the console (uses Java Util Logging levels. The default is _INFO_.
+| HAWKULAR_APM_LOG_LEVEL | The logging level to use when reporting client collector information to the console (uses Java Util Logging levels. The default is _INFO_. Allowed values are FINEST, FINER, FINE, INFO, WARNING and SEVERE.
```

Comment:
LGTM, 

minor comment: maybe use the same styling? In the previous sentence it is in italics..


Just for curiosity. Did you looked/used `setenv.sh` ? There is a a partial list of all values